### DISCUSSION TITLE
Google assistant in Bromite?

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#1882] [Q&A] [answered]
![Screenshot](https://user-images.githubusercontent.com/95953932/158027648-a5d11488-4196-4189-98b0-3bc7067da5af.jpg)

why does Bromite have google assistant feature?

**Comment by csagan5** [comment, marked as answer]
It does not have it, it's inert. Will be removed on next release.

**Reply by uazo** [reply]
we could even think about deleting it completely, it would save some space and maybe even some compilation time:

![image](https://user-images.githubusercontent.com/29201891/158049862-c8a9bb38-2646-42c2-8879-02734a13b0ea.png)

**Reply by csagan5** [reply]
It is not a good idea, see https://github.com/bromite/bromite/pull/1901

**Comment by ghost** [comment]
Bromite also has the Translate Sites feature in the newer version, It did not have the feature in older versions.

**Reply by csagan5** [reply]
Where do you see that?

**Reply by ghost** [reply]
> Where do you see that?

![Screenshot_20220319-165054](https://user-images.githubusercontent.com/95953932/159119027-8b5f9c86-2879-476a-8b9a-0947a841177c.jpg)
![Screenshot_20220319-165058](https://user-images.githubusercontent.com/95953932/159119038-3941b8cc-44b4-410d-b182-3270e438d1c6.jpg)

**Reply by csagan5** [reply]
These are also non-working and can be removed.

**Comment by ghost** [comment]
Bromite has new google feature on latest version and should be removed
![Screenshot_20220401-021115-1](https://user-images.githubusercontent.com/102543391/161145990-69ba8e45-124d-404f-b086-8e736adee383.jpg)

**Reply by csagan5** [reply]
It will be removed as well.

**Reply by ghost** [reply]
Okay

**Comment by ghost** [comment]


**Reply by csagan5** [reply]
No, there is no need to mention anyone. Whoever is following the discussions will receive notifications from GitHub.
